src/ClimaAtmos/Dycore/src/VanillaAtmosDiscretizations_kernels.jl

```diff
@@ -35,25 +49,39 @@ function volumegrad!(::Val{2}, ::Val{N}, ::Val{nmoist}, ::Val{ntrace},
   s_u = Array{DFloat}(undef, Nq, Nq)
   s_v = Array{DFloat}(undef, Nq, Nq)
   s_T = Array{DFloat}(undef, Nq, Nq)
+  s_θ = Array{DFloat}(undef, Nq, Nq)
 
+  q_m = zeros(DFloat, max(3, nmoist))
+    
```

Comment:
Though it does not render, here and elsewhere there is trailing whitespace. Not a big deal, but cleaner if it's removed. This can be done with sed in both the `src` and `drivers` directory:
```
sed -i '' -E 's/[ '$'\t'']+$//' *.jl
```